elixir/apps/domain/priv/repo/migrations/20250508192722_add_indexes_to_actor_group_memberships.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Domain.Repo.Migrations.AddIndexesToActorGroupMemberships do
+  use Ecto.Migration
+
+  @disable_ddl_transaction true
+  @disable_migration_lock true
```

Comment:
Added these based on this article from fly.io:
https://fly.io/phoenix-files/migration-recipes/#adding-an-index

Reply:
I also saw this in the docs:

```
Alternately, you can add @disable_migration_lock true to your migration file. 
This would mean that different nodes in a multi-node setup could run the same 
migration at once.  It is recommended to isolate your migrations to a single node 
when using concurrent index creation without an advisory lock.
```

@jamilbk Do we lock our migrations to a single node?

Reply:
We do use advisory locks now right? So I'm not sure this is needed?

See the main docs at https://hexdocs.pm/ecto_sql/Ecto.Migration.html#index/3-adding-dropping-indexes-concurrently

Reply:
Ah, ok, so only the `@disable_ddl_transaction true` is needed?

Reply:
Let's see! 🤠